### PR TITLE
Bill/manual testing

### PR DESCRIPTION
**Pull request by LivingOnEdge**

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -1119,6 +1119,48 @@ These instructions only provide a starting point for testers to work on; testers
...
+
+. Editing a meme while all memes are listed
+
+.. Prerequisites: Weme is in the memes tab.
+.. Test case: `edit 1 d/new description t/tag` +
+   Expected: The meme in the given index is edited. The view is updated to reflect this change. The status message states the edit was successful.
+.. Test case: `edit 1` +
+   Expected: No meme is edited. Weme prompts the user to specify at least one field to be edited.
+.. Other incorrect add commands to try: `edit`, +
+   Expected: Similar to previous.
```
Comment (cutieprobe) [inline docs/DeveloperGuide.adoc]: The behaviour should be weme prompting invalid command format, which is not the same as the previous one

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -1119,6 +1119,48 @@ These instructions only provide a starting point for testers to work on; testers
...
+
+. Editing a meme while in `find`
+
+.. Prerequisites: Weme is in the memes tab and just executed a `find` command.
+.. Similar to above.
```
Comment (gary-lgy) [inline docs/DeveloperGuide.adoc]:
```suggestion
.. Similar to above. Weme will edit the meme specified by the index in the displayed index.
```

File: docs/DeveloperGuide.adoc [hunk 3]
```diff
@@ -1119,6 +1119,48 @@ These instructions only provide a starting point for testers to work on; testers
...
+
+=== Deleting a meme
+
+. Deletes a meme while all memes are listed
```
Comment (gary-lgy) [inline docs/DeveloperGuide.adoc]:
```suggestion
. Deleting a meme while all memes are listed
```

File: docs/DeveloperGuide.adoc [hunk 4]
```diff
@@ -1119,6 +1119,48 @@ These instructions only provide a starting point for testers to work on; testers
...
+
+.. Prerequisites: Weme is in the memes tab.
+.. Test case: `delete 1` +
+   Expected: The meme in the given index is deleted. Meme is removed from the view. The status message states the delete was successful.
+.. Test case: `delete 0` +
+   Expected: No meme is deleted. Weme prompts the user to give a valid index.
+.. Other incorrect add commands to try: `delete`, `delete x` (where x is larger than the list size) +
```
Comment (gary-lgy) [inline docs/DeveloperGuide.adoc]:
```suggestion
.. Other incorrect delete commands to try: `delete`, `delete x` (where x is larger than the list size) +
```

File: docs/DeveloperGuide.adoc [hunk 5]
```diff
@@ -1119,6 +1119,48 @@ These instructions only provide a starting point for testers to work on; testers
...
+=== Editing a meme
+
+. Editing a meme while all memes are listed
+
+.. Prerequisites: Weme is in the memes tab.
+.. Test case: `edit 1 d/new description t/tag` +
+   Expected: The meme in the given index is edited. The view is updated to reflect this change. The status message states the edit was successful.
+.. Test case: `edit 1` +
+   Expected: No meme is edited. Weme prompts the user to specify at least one field to be edited.
+.. Other incorrect add commands to try: `edit`, +
```
Comment (gary-lgy) [inline docs/DeveloperGuide.adoc]:
```suggestion
.. Other incorrect edit commands to try: `edit`, +
```

File: docs/DeveloperGuide.adoc [hunk 6]
```diff
@@ -1119,6 +1119,48 @@ These instructions only provide a starting point for testers to work on; testers
...
+   Expected: The meme in the given index is deleted. Meme is removed from the view. The status message states the delete was successful.
+.. Test case: `delete 0` +
+   Expected: No meme is deleted. Weme prompts the user to give a valid index.
+.. Other incorrect add commands to try: `delete`, `delete x` (where x is larger than the list size) +
+   Expected: Similar to previous.
+
+. Deleting a meme while in `find`
+
+.. Prerequisites: Weme is in the memes tab and just executed a `find` command.
+.. Similar to above.
```
Comment (gary-lgy) [inline docs/DeveloperGuide.adoc]:
```suggestion
.. Similar to above. Weme will delete the meme specified by the index in the displayed list.
```

File: docs/DeveloperGuide.adoc [hunk 7]
```diff
@@ -1119,6 +1119,48 @@ These instructions only provide a starting point for testers to work on; testers
...
+
+=== Clearing memes
+
+. Clearing all memes in the memes tab
+
+.. Prerequisites: Weme is in the memes tab.
+.. Test case: `clear` +
+   Expected: All memes in the memes tab has been cleared. The status message states the clearing was successful.
+
```
Comment (gary-lgy) [inline docs/DeveloperGuide.adoc]: Add a note to say that they can undo this afterwards.

File: docs/DeveloperGuide.adoc [hunk 8]
```diff
@@ -1232,7 +1274,7 @@ Similar to like. Command is `dislike`.
 === Staging a Meme
 
 . Stages a meme while all memes are listed.
```
Comment (gary-lgy) [inline docs/DeveloperGuide.adoc]:
```suggestion
. Staging a meme while all memes are listed.
```
Standardise all the tenses please.

**Pull request merged**